### ISSUE TITLE
Throwing Sand To Blind Enemies

### ISSUE THREAD

**Issue by dominic-dimico** [CleverRaven/Cataclysm-DDA#74556]
### Is your feature request related to a problem? Please describe.

It's very intuitive how cameras could blind zombies.  I figured sand would blind anything for a few seconds, as it does in real life (and movies!).  However, when I tried to throw sand at zombies, it just falls to the floor.  As a player, I think it would be an interesting alternative to a camera for blinding enemies, and not too overpowered since it requires finding it, carrying it around, and switching out any equipped weapons to throw it.  It would be ideal for martial artists as an opening move in a battle; or as an escape tactic from a single, powerful enemy.

### Solution you would like.

Could it be possible to produce the blind effect when sand is thrown at an enemy?  It seems like it would be on the easier side to implement since the code for the blindness effect is already there.

### Describe alternatives you have considered.

As a player, I've thought about throwing cayenne pepper, too.  In real life, that stuff has the ability to blind anything with eyes and stun anything that breathes, but it can't be thrown far.

### Additional context

I've played to the late game, and I appreciate how real-life logic dictates the battle mechanics, and is used so often to solve problems.  Maybe this could be one of those solutions.

**Comment by ltngonnguyen** [comment]
Agreed, we already have this kind of tactic by using the tear gas spray to create a tile of tear gas in a narrow corridor and funnel zombies there for easy whooping, can't imagine how much more difficult it could be to throw some sand at individual enemies.

**Comment by Kantonine** [comment]
We be learning from the Great Khans with dis one!

**Comment by ADekema** [comment]
I doubt that it would do much against zombies given that they don´t care about irretants and such. Maybe it could make thier eyes watery and vision blurry which would reduce sight radius and induce a accuracy penalty. It would work on normal animals, mutants and (feral) humans though. Maybe not insects as they have compound eyes.

**Comment by TheSaddestGoomba** [comment]
Spray paint cans would be another candidate for this.
